server/sonar-server/src/test/java/org/sonar/server/es/BulkIndexerTest.java

```diff
@@ -102,6 +114,42 @@ public void bulk_delete() throws Exception {
     assertThat(count()).isEqualTo(removeFrom);
   }
 
+  @Test
+  @Ignore
+  public void when_index_is_done_EsQueues_must_be_deleted() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)